spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -530,15 +550,33 @@ helm install --name my-release \
 [[getting-started-deploying-streams]]
 == Deploying Streams
 
-This section covers how to deploy streams with Spring Cloud Data Flow.
-Your choice at this point is whether to use Skipper, which lets you discover applications and manage their lifecycles. For more about Skipper, see https://cloud.spring.io/spring-cloud-skipper/.
+This section covers how to deploy streams with Spring Cloud Data Flow and Skipper. Later lets you discover applications and manage their lifecycles.
```

Comment:
should `and Skipper. Later lets you` be something like `and Skipper which lets you` ?

Reply:
removed